pkg/monitor/logrecord.go

```diff
@@ -167,7 +159,6 @@ type LogRecordNotifyVerbose struct {
 	DstIdentity      uint64                     `json:"dstIdentity"`
 	Verdict          accesslog.FlowVerdict      `json:"verdict"`
 	HTTP             *accesslog.LogRecordHTTP   `json:"http,omitempty"`
-	Kafka            *accesslog.LogRecordKafka  `json:"kafka,omitempty"`
```

Comment:
This is technically a breaking change in the monitor API. Not opposed to this, but just wanted to point that out explicitly, e.g. legacy versions of Hubble (v0.5) would not be able to recognize Kafka records after this (we don't support Hubble v0.5 on Cilium 1.9, so that's fine).

Reply:
Good to know, will leave this for further discussion.

Reply:
I think it's fine to go through with that, but would be nice to have a release note. If it's necessary to leave this, can we have information about Kafka in two places (AFAIU right now it will be stored in `L7` field?).

Reply:
I'll check how hard it would be to keep using `LogRecordKafka`, will report back!